SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeploy.java

```diff
@@ -81,7 +82,8 @@ public SingularityDeploy(@JsonProperty("requestId") String requestId,
       @JsonProperty("loadBalancerGroups") Optional<List<String>> loadBalancerGroups,
       @JsonProperty("considerHealthyAfterRunningForSeconds") Optional<Long> considerHealthyAfterRunningForSeconds,
       @JsonProperty("loadBalancerOptions") Optional<Map<String, Object>> loadBalancerOptions,
-      @JsonProperty("skipHealthchecksOnDeploy") Optional<Boolean> skipHealthchecksOnDeploy) {
+      @JsonProperty("skipHealthchecksOnDeploy") Optional<Boolean> skipHealthchecksOnDeploy,
+      @JsonProperty("healthCheckScheme") Optional<String> healthcheckScheme) {
```

Comment:
hate to nitpick but i think `healthcheckProtocol` is more accurate. also considering the fact that we only support http(s), might want to make it an enum instead